cobalt/browser/h5vcc_experiments/public/mojom/h5vcc_experiments.mojom

```diff
@@ -0,0 +1,34 @@
+// Copyright 2025 The Cobalt Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+module h5vcc_experiments.mojom;
+
+import "mojo/public/mojom/base/values.mojom";
+
+enum OverrideState {
+  OVERRIDE_USE_DEFAULT,
```

Comment:
Document these values

Reply:
Also mention that these replicate existing ones: https://source.chromium.org/chromium/chromium/src/+/main:base/feature_list.h;l=296-299;drc=eb76903b3b199d9464c7a61abe02af0f329e1a74

Question: Can we just do 
```
[Native]
enum OverrideState;
```
? 

https://chromium.googlesource.com/chromium/src/+/main/docs/mojo_ipc_conversion.md#repurposing-and-invocations

Reply:
Seems like IPC_ENUM_TRAITS is deprecated. And I found this
https://groups.google.com/a/chromium.org/g/chromium-dev/c/bhy8P7tPE4I/m/MFKwNofiCAAJ which suggests defining the enum in .mojom directly

Reply:
Didn't mean to look at that documentation specifically (it was targeting
the IPC-to-Mojo migration, just the [Native] entry details), apologies.
I read the discussion but the usage seems pervasive, e.g. [media types](https://source.chromium.org/chromium/chromium/src/+/main:media/mojo/mojom/media_types.mojom). I see it basically as a shortcut to avoid having to codify the
traits (from this mojo to the FeatureList enum and viceversa).

Reply:
Can we unblock this PR first? I added a TODO for this. We can take the discussion there